docs/software/wifi_hotspot.md

```diff
@@ -2,12 +2,80 @@
 
 ## Overview
 
-- [WiFi HotSpot - Turn your device into a wireless hotspot/access point.](https://dietpi.com/phpbb/viewtopic.php?p=1207#p1207)  
-- [Tor HotSpot - Optional: Routes all WiFi HotSpot traffic through the Tor network.](https://dietpi.com/phpbb/viewtopic.php?p=1529#p1529)  
+- [**WiFi HotSpot - Turn your device into a wireless hotspot/access point**](#wifi-hotspot_1)  
+- [**Tor HotSpot - Optional: Routes all WiFi HotSpot traffic through the Tor network**](#tor-hotspot)  
+
+??? info "How do I run **DietPi-Software** and install **Optimised software** ?"
+    To install any of the **DietPi Optimised Software** listed below run from the command line:
+    ```
```

Comment:
All sections with \`\`\` should have an empty line above and below?

Reply:
This will be corrected when we change the info box by deleting the `dietpi-launcher` remark.